live-site/components/hacking-remaning/HackingRemaining.styles.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+import { colors } from '../../../shared-ui/style/colors';
+
+const StyledHackingRemainingContainer = styled.div`
+  z-index: 20;
+  float: right;
+  background-color: ${colors.BUTTON_DARK_GREEN};
+
+  border-radius: 1em;
+  border: 0.3em solid ${colors.BUTTON_DARK_GREEN};
+  filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25))
+    drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+`;
+
+const StyledHackingRemainingHeader = styled.p`
```

Comment:
<img width="268" alt="Screenshot 2022-11-23 at 11 01 24 AM" src="https://user-images.githubusercontent.com/59738880/203592588-2ee69073-40c2-42aa-bf20-6406977e3ed4.png">
According to the design, this is split up on 2 different lines so lets set a fixed width / max width or something so it's always 2 lines

And you'll also want to apply a text align center to center the text